src/Composers/TimezoneLocaleComposer.php

```diff
@@ -17,10 +18,12 @@ class TimezoneLocaleComposer
      */
     public function compose(\Illuminate\View\View $view)
     {
-        $langs = array_map(function ($lang) {
+        $enabledLangs = Config::get('langs');
```

Comment:
NB, we should generally try to avoid regretting Config:: and Setting:: within our code. The view composer shouldn't know about how we store the config.